tox.ini

```diff
@@ -5,7 +5,7 @@ envlist =
     py38-django{31}
 
 [testenv]
-commands = pytest --cov=rest_live --cov-append --junitxml=test-results/tests.xml
+commands = pytest --cov=rest_live --cov-append --cov-report term-missing:skip-covered --junitxml=test-results/tests.xml
```

Comment:
Small tweak to display the line numbers missing coverage, makes it easier to track untested code